drivers/include/kw2xrf.h

```diff
@@ -130,6 +131,13 @@ typedef struct {
                                              this is required to know when to
                                              return to @ref kw2xrf_t::idle_state */
     int16_t tx_power;                   /**< The current tx-power setting of the device */
+    bool    ack_requested;              /**< ACK was requested for last frame */
+    bool    ch_clear;                   /**< CCA indicated channel clear */
+    bool    waiting_for_cca;            /**< Indicate whether CCA is still ongoing */
+    bool    tx_done;                    /**< Indicate whether TX completed */
+    bool    ack_rcvd;                   /**< Indicate if ACK was received for last transmission */
+    bool    cca_before_tx;              /**< true if CCA shall be performed before TX */
+    bool    tx_cca_pending;             /**< true a manual CCA was started and a TX should be triggered on channel clear indication */
```

Comment:
```suggestion
    bool    ack_requested : 1;          /**< ACK was requested for last frame */
    bool    ch_clear : 1;               /**< CCA indicated channel clear */
    bool    waiting_for_cca : 1;        /**< Indicate whether CCA is still ongoing */
    bool    tx_done : 1;                /**< Indicate whether TX completed */
    bool    ack_rcvd : 1;               /**< Indicate if ACK was received for last transmission */
    bool    cca_before_tx : 1;          /**< true if CCA shall be performed before TX */
    bool    tx_cca_pending : 1;         /**< true a manual CCA was started and a TX should be triggered on channel clear indication */
```

How much more ROM would this need? should save a few bytes of RAM.

Reply:
IIRC there's still the discussion regarding portability of bitfields. Is there any outcome? @kaspar030

Reply:
This structure is not send over the wire/air so no need to worry about portability

Reply:
anyway, it reduces 8 bytes of RAM and adds 64 bytes of ROM